components/utilities/utility-icon/index.jsx

```diff
@@ -62,9 +66,11 @@ const UtilityIcon = (
 		modifiedPath = `${context[`${category}Sprite`]}#${name}`;
 	} else {
 		// Otherwise, use external URLs for icons
+		const svgAssetName =
+			direction === DIRECTIONS.RTL ? 'symbols.svg' : 'symbols.svg'
```

Comment:
The changes in this file are for illustration purposes only. This L70 was originally set to 

direction === DIRECTIONS.RTL ? 'symbols-rtl.svg' : 'symbols.svg'

However, I later realized during manual testing that the icons from symbols-rtl.svg is no different from if I follow the SLDS guideline below 

![Screen Shot 2019-06-27 at 4 27 35 PM](https://user-images.githubusercontent.com/4957585/60307224-8ad46f00-98f8-11e9-8d14-e01f60a80b30.png)

Hence, the actual intended changes are in components/icon/index.jsx, and we do not seem to need to the rtl svgs at all.

I will revert all the changes in this file upon review approval.

Reply:
and of course, I copied the rtl svgs to my local repo for testing

Reply:
This is wrong. We do **not** want to flip every icon. Some icons do not have to be flipped and some cannot be flipped. There is a list of icons which UX white-listed for flipping and they are flipped in the RTL sprite. Please use the RTL sprite instead.

https://docs.google.com/presentation/d/1k7UsZg2PSJ6p-BB6BV9jQ82fiXscqfP5EoKnVSINnZ4/edit?ts=5c54c471#slide=id.g4e0eb6c79e_0_0

https://docs.google.com/spreadsheets/d/1IAS0T6n4IsE3V4awovcEzd62GOfcsQtir5VLuPiQouY/edit?ts=5c655e50#gid=0

`slds-icon_flip` can be applied on a case by case basis, but it cannot be bulk-applied to all icons.

Reply:
Should we introduce an array into code to document which ones get flipped and use the RTL sprite?

This seems like something we could ask SLDS to have in their design tokens file? Temporarily, I allow this list in this library until we can get something from SLDS.

Reply:
The doc is helpful. Let me compare the the icon appearance using two approaches (slds-icon_flip versus rtl svg). And yes, I can introduce an array storing the names of the icons that needs to flipped for now...

Reply:
It would be better if SLDS had this documented on their side and we could just use the RTL sprite. An array will quickly get out-of-sync.

Reply:
I wholeheartedly agree.

Reply:
@minevskiy @interactivellama Please review again. If we really want to limit the usages of rtl sprites based on icon names in addition to the rtl context flag, that can be done in a follow-up issue fix when those icons are mentioned in slides design tokens or somewhere else.

Reply:
temporarily added assets/icons/standard-sprite/svg/symbols-rtl.svg in this pr to be used for automation purpose since rtl sprites are not available in the current dsr master branch. Once SLDS version updates in DSR, this file can be removed.

Reply:
I’m cool with the file temporarily. We have to note the tech debt though for when the image snapshots are upgraded to 2.9. Please comment on #2080

Reply:
Sure, just left a comment there.

Reply:
@interactivellama The following command is failing on travis CI, but I doubt it's a real issue.

Earlier I ran `npm run lint` as well as `npm run test:stories`, all tests passed. Not sure what this `npm run lint != SNAPSHOT_TESTS` is doing.

COMMAND: npm run lint != SNAPSHOT_TESTS: npm run test:stories

Reply:
@interactivellama Can you review this pr again and let me know if there are any further comments?